docs/courses/design/personas.md

```diff
@@ -15,60 +15,51 @@ e.g.,
 novice understanding of R but advanced understanding of medical imaging
 (or vice versa).
 
-## Advanced Alex
+## Unaware Umberto
 
-- Age: 34
-- Location: Reno, Nevada, USA
-- Education: Bachelor in Mathematics and Master in Finance & Risk Management - University of Nevada
-- Personal: Interested in teaching courses himself some day.
+- Age: 27
+- Location: London, UK
+- Education: Bachelor in Philosophy & Bachelor in Business  Economics - King's College London
+- Personal: Partially deaf, so prefers materials with closed captioning.
 
-**Previous Jobs**
+**Previous Job**
 
-- Pricing analyst for power distributor.
-- Financial analyst at Bank of America.
+- Financial advisor at Lloyds Banking Group.
 
 **Current Job**
 
-- Head quantitative analyst in small hedge fund.
-
-**Career Goal**
-
-After developing his skill set and finding out what he likes in
-previous jobs, Alex is set on staying in this role for quite a
-while. He was recently made part of the management team. His main
-concern is driving the company forward and outpacing competitors.
+- Financial analyst at KPMG.
```

Comment:
Doesn't this count as domain expertise? I'd be more tempted to make Umberto a recent graduate, or give him a totally non-technical background. Perhaps he was a DJ until his hearing problems caused him to consider a new career.